tendrl_gluster_peers_bricks.yml

```diff
@@ -1,9 +1,10 @@
 ---
 #
-# * Install Tendrl server and tendrl-gluster nodes.
-# * Configure gluster trusted storage pool and create bricks 
-#   on all available devices on gluster nodes.
+# Install Tendrl server and tendrl-gluster nodes.
+# Configure gluster trusted storage pool and create bricks
+# on all available devices on gluster nodes.
 #
 - include: qe_pre_installation_tasks.yml
-- include: tendrl_gluster.yml
```

Comment:
My opinion is, that `tendrl_gluster.yml` playbook is redundant and should be removed, but I would like to keep it there for few days for backward compatibility with jenkins jobs.